known-issues.yml

```diff
@@ -21,6 +21,12 @@ pattern-ZibHealthProfessionalReference:
     StructureDefinition:
       - message: "sd-pg-08: 'The title of the StructureDefinition should conform to the profiling guidelines'"
         reason: The title is shown in the hosting profiles as the name of the datatype. "zib HealthProfessional Reference" is more descriptive than the title according to the profiling guidelines ("pattern ZibHealthProfessionalReference").
+        
+zib-AbilityToUseToilet:
+  zib deviations:
+      Observation:
+      - short: AbilityToUseToilet
+        reason: The root concept name is an mistake in the zib and will be fixed at the next pre-publication
```

Comment:
I was not directly sure what you meant here. Can you include the wrong name 'ToiletUseAbility'  here too?